test/appium/views/base_element.py

```diff
@@ -151,7 +151,7 @@ def swipe_element(self):
         location, size = element.location, element.size
         x, y = location['x'], location['y']
         width, height = size['width'], size['height']
-        self.driver.swipe(start_x=x + width * 2, start_y=y + height / 2, end_x=x, end_y=y + height / 2)
+        self.driver.swipe(start_x=x + width, start_y=y + height / 2, end_x=x, end_y=y + height / 2)
```

Comment:
please use half of the screen width instead of element width, e.g self.driver.get_window_size() so the method will work for any element